### DISCUSSION TITLE
RPR SDK output blend RAW with Depth

### DISCUSSION THREAD

**Discussion by Habaybeh** [GPUOpen-LibrariesAndSDKs/RadeonProRenderBlenderAddon#521] [Q&A]
Hey everyone, I'm trying to use RPR Addon for Blender to export 1 frame as .RPR file and then render it using RPR SDK

The file renders successfully but outputs the result file in a raw view, and a .depth file. Meaning that the result file is a bit dark. Although when I render it within Blender it renders successfully (aka Standard view transform).

I'm trying to get a combined file when I render using the SDK, the same as I get using Blender. Is there a flag or a config key that does this?

The attached file RPR Blender is how I get it using RPR within Blender, and the other file is the render result using the SDK.

P.S: I had to post here since I couldn't find a discussion section on RPR SDK's Git repository

![RPR Blender](https://user-images.githubusercontent.com/31856652/173152859-55f3f1c4-564b-4b29-9a4e-9697650ccc0b.png)
![RPR SDK](https://user-images.githubusercontent.com/31856652/173152866-5d286efa-97f8-45d3-a0ba-c27c596c1162.png)

**Comment by bsavery** [comment]
You can do the color management manually yourself in Photoshop or really any photo editing util.  Or use Blender's compositor using the color space node https://docs.blender.org/manual/en/dev/compositing/types/converter/color_space.html

The render utility is really meant to feed into some color management or compositing utility.  In fact this is the same output you get from RPR in Blender, it's just converted in space by blender when read in.
